funnel/templates/macros.html.jinja2

```diff
@@ -106,10 +106,79 @@
     {%- if project.sessions %}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'schedule' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'schedule' -%}{{ project.url_for('schedule') }}{%- else %}#schedule{%- endif %}">Schedule</a>{%- endif %}
     {%- if project.hasjob_embed_url -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'project' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#related-jobs">Jobs</a>{%- endif %}
     {%- if project.primary_venue -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'project' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#venue">Venue</a>{%- endif %}
-    {%- if project.proposals -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'project' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#proposals">Proposals</a>{%- endif %}
+    {%- if project.proposals -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'proposals' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'proposals' -%}{{ project.url_for('view_proposals') }}{%- endif %}#proposals">Proposals</a>{%- endif %}
```

Comment:
What is `current_page`? We can do this as `current_view.name` and make sure all the views have a `name` attribute.

Reply:
This macro function renders header for 2 types of pages - project and schedule. `current_page` is passed to the function from templates. It's not exactly the name of the view. E.G. SessionView's `view` and ProjectScheduleView's `view` both use same `schedule.html.jinja2` template which pass `schedule` to this function. It's more like view type than view name.